labgrid/driver/dediprogflashdriver.py

```diff
@@ -0,0 +1,72 @@
+import os.path
+import logging
+import attr
+
+from ..resource import NetworkDediprogFlasher
+from ..factory import target_factory
+from ..step import step
+from .common import Driver, check_file
+from ..util.managedfile import ManagedFile
+from ..util.helper import processwrapper
+
+
+@target_factory.reg_driver
+@attr.s(eq=False)
+class DediprogFlashDriver(Driver):
+    """ The DediprogFlashDriver uses the dediprog utility to write an image
+    to a raw rom. The driver is a pure wrapper of the dpcmd utility"""
+    bindings = {
+        'dediprog_resource': {"DediprogFlasher", NetworkDediprogFlasher},
+    }
+
+    image = attr.ib(default=None)
+
+    def __attrs_post_init__(self):
+        super().__attrs_post_init__()
+        self.logger = logging.getLogger(f'{self}')
+        if self.target.env:
+            self.tool = self.target.env.config.get_tool('dpcmd')
+        else:
+            self.tool = 'dpcmd'
+        self.logger.debug('Tool %s', self.tool)
```

Comment:
```suggestion
```
I don't think this is useful, `processwrapper.check_output()` logs the command anyway and passes it to the StepReporter.